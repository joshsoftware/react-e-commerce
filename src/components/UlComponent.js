import React from 'react';
import PropTypes from 'prop-types';

const UlComponent = ({ liElement }) => {
  return <ul className="list-inline">{liElement}</ul>;
};

export default UlComponent;

UlComponent.propTypes = {
  liElement: PropTypes.oneOfType([PropTypes.element, PropTypes.array])
};