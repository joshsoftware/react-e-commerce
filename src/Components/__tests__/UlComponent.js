import React from 'react';
import { render } from '@testing-library/react';
import UlComponent from '../UlComponent';

describe('UlComponent ', () => {
  it('Must return entire UnorderdListcontainer', () => {
    const { asFragment } = render(<UlComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
