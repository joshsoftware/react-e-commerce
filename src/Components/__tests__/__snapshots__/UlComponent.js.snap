// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UlComponent  Must return entire UnorderdListcontainer 1`] = `
<DocumentFragment>
  <ul
    class="list-inline"
  />
</DocumentFragment>
`;
